Src/BootCamp.Chapter/LinqDemo.cs

```diff
@@ -6,55 +6,75 @@ namespace BootCamp.Chapter
 {
     internal class LinqDemo
     {
+        private static List<int> list1 = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+        private static List<int> list2 = new List<int> { 1, 11, 12 };
+        private static List<int> list3 = new List<int> { 4, 2, 5, 4 };
+
         internal static void Demo()
         {
-            var list1 = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
-            var list2 = new List<int> { 1, 11, 12 };
-            var list3 = new List<int> { 4, 2, 5, 4 };
-
-            // Any. so Im going to check if any number in list1 is even.
-            var valid = list1.Any(n => n % 2 == 0);
-
-            if (valid)
-            {
-                Console.WriteLine($"The list does contain a even number{Environment.NewLine}");
-            }
-            else
-            {
-                Console.WriteLine("The list does not contain a even number");
-            }
+            AnyDemo(list1);
+            CountDemo(list1);
+            SortDemo(list3);
+            WhereDemo(list1);
+            UnionDemo(list1, list2);
+            IntersectionDemo(list1, list2);
+            ExtractDemo(list1, list2);
+        }
 
-            // Count. So we are going to count how many even numbers there are in list1. 
-            var count = list1.Count(n => n % 2 == 0);
-            Console.WriteLine($"There are {count} even numbers in list1{Environment.NewLine}");
+        private static void ExtractDemo(List<int> list1, List<int> list2)
+        {
+            Console.Write($"The intersection of a list and another list looks like this: ");
```

Comment:
The demos are not very high effort. You copy pasted writes and forgot to even update the messages.

Reply:
In the future try to come up with more specific concepts: like people and pets or similar. Now it's just a meaningless list.